process/peer/process_test.go

```diff
@@ -596,6 +598,535 @@ func TestValidatorStatisticsProcessor_UpdatePeerStateCallsIncrease(t *testing.T)
 	assert.True(t, increaseValidatorCalled)
 }
 
+func TestValidatorStatisticsProcessor_UpdatePeerState_IncreasesConsensusPreviousMetaBlock_SameEpoch(t *testing.T) {
+	t.Parallel()
```

Comment:
Add empty line after each t.Parallel() in this file

Reply:
fixed